draft-irtf-cfrg-hpke.md

```diff
@@ -1642,6 +1642,11 @@ modes. This is because ephemeral randomness is used on the sender's side, which
 is supposed to be erased directly after computation of the KEM shared secret and
 ciphertext.
 
+HPKE ciphertexts are not forward-secure with respect to receipient compromise. 
+In the Base and Auth modes, a given ciphertext can be decrypted if the recipient's 
+private decryption key is compromised. In the PSK and AuthPSK modes, a given ciphertext
+can be decrypted if the recipient's private key and the PSK are compromised.
+
```

Comment:
```suggestion
```

Uh, I am sorry about this, it seems I messed up the merge from master into my last pull request (offending commit: https://github.com/cfrg/draft-irtf-cfrg-hpke/commit/2a1d2da03424d82946775314c0553bf7ab0bf4e9).

I think we should just delete this paragraph. Almost the same text is already in 9.1, and the two paragraphs here are more precise.

Additional suggestion: In 9.1 in the paragraph about forward secrecy, we could link to 9.7.4 directly instead of only to 9.7.

Reply:
Works for me, too!